CIFAR10/train/fastai_pytorch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/",
+    "model": "Custom Wide Resnet",
```

Comment:
Could you expand on this a bit more? I believe wide resents are parameterized - what parameters do you use for your custom wide resnet?

Reply:
linked to the model we are using!